locales/enUS/translation.json

```diff
@@ -337,6 +341,7 @@
     "passwordMin": "Passphrase must be at least 8 characters in length.",
```

Comment:
I updated the code, changing "passphrase" to "password" -- an update seems to be needed for this PR.